apps/verticalhome/js/stores/item.js

```diff
@@ -217,8 +218,10 @@
           self.gridOrder = configurator.getGrid();
           self.populate(cb);
         });
-        window.addEventListener('configuration-pin-app-ready', function onPinAppReady() {
-          window.removeEventListener('configuration-pin-app-ready', onPinAppReady);
+        window.addEventListener('configuration-pin-app-ready',
```

Comment:
'config-pin-app-ready' name might work, but I would say that we can also keep it as it is done here. Not need to cry on this :)